tests/meta/test_adaptive_random_forest_regressor.py

```diff
@@ -116,9 +116,9 @@ def test_adaptive_random_forest_regressor_perceptron():
     error2 = mean_absolute_error(y_true, y_pred2)
     error3 = mean_absolute_error(y_true, y_pred3)
 
-    expected_error1 = 118.69
-    expected_error2 = 121.56
-    expected_error3 = 117.96
+    expected_error1 = 126.01
```

Comment:
I'm wondering why the expected error changed?

Reply:
While simplifying the node structure we found a minor bug in the `ARFHoeffdingTreeRegressor` nodes. That's why the results slightly changed

Reply:
I see, thanks @smastelini